modules/compiler/builtins/scripts/generate_header_30.sh

```diff
@@ -391,6 +391,8 @@ function all_atomic()
     for i in int long uint ulong float double
     do
       double_support_begin $i
+      echo "void __CL_BUILTIN_ATTRIBUTES atomic_store(volatile $k atomic_$i *object, $i desired);"
```

Comment:
I believe we test some of the others, is it possible to add a test for these ones just so it's at least covered?

Reply:
Done, I changed the existing `atomic_store_explicit` tests to use any of the three versions so that they all get used.